projects/addon-editor/directives/tiptap-editor/tiptap-editor.directive.ts

```diff
@@ -0,0 +1,31 @@
+import {Directive, Inject, Input, Output} from '@angular/core';
+import {TuiEditor} from '@taiga-ui/addon-editor/abstract';
+import {TIPTAP_EDITOR_PROVIDERS} from './tiptap-editor.providers';
+import {TuiTiptapEditorService} from './tiptap-editor.service';
+
+@Directive({
+    selector: '[tuiTiptapEditor]',
+    exportAs: 'tuiDesignMode',
+    providers: [TIPTAP_EDITOR_PROVIDERS, TuiTiptapEditorService],
+})
+export class TuiTiptapEditorDirective {
+    @Input()
+    set value(value: string) {
+        if (value !== this.editor.html) {
+            this.editor.setValue(value);
+        }
+    }
+
+    @Input()
+    set editable(editable: boolean) {
+        this.editor.editable = editable;
+    }
+
+    @Output()
+    valueChange = this.editor.valueChange$;
+
+    @Output()
+    stateChange = this.editor.stateChange$;
+
+    constructor(@Inject(TuiTiptapEditorService) public editor: TuiEditor) {}
```

Comment:
private readonly maybe?

Reply:
No, I need editor in parent wrapper to checking some states or editing links (

Reply:
Then just readonly ;)